pkg/libvirttools/virtualization.go

```diff
@@ -139,8 +140,9 @@ func (v *VirtualizationTool) processVolumes(mounts []*kubeapi.Mount, domXML stri
 	}
 
 	for _, mount := range mounts {
+		volumeName := containerName + "_" + strings.Replace(*mount.ContainerPath, "/", "_", -1)
```

Comment:
mount.ContainerPath is a pointer, so doing `*` when it's nil should just fail/panic

You may of course do manual checks, but these `GetSomething` methods in CRI are already doing that and returning empty structs or strings when pointers are null.

I know that we have a lot of places we are not using these methods, shame on me, I'm going to change them all soon.